appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/AssetList.java

```diff
@@ -110,17 +110,19 @@ private void refreshAssetList() {
               nodeName.length());
 
         String fileSuffix = node.getProjectId() + "/" + node.getFileId();
-        String treeItemText = "<span style='cursor: pointer'>" + nodeName + "</span>";
+        String treeItemText = "<span style='cursor: pointer'>";
+        Image mediaIcon = null;
```

Comment:
Alternatively, instead of setting and checking the variable, within the if-else blocks use `treeItemText += new Image(...);` since at most only one path will be taken.